test/Microsoft.ML.Tests/OnnxSequenceTypeTest.cs

```diff
@@ -64,5 +82,21 @@ public void OnnxSequenceTypeWithColumnNameAttributeTest()
             }
 
         }
+
+        private static PredictionEngine<FloatInput, ProblematicOutputObj> CreatePredictorWithPronlematicOutputObj()
+        {
+            var onnxModelFilePath = Path.Combine(Directory.GetCurrentDirectory(), "zipmap", "TestZipMapString.onnx");
+
+            var ctx = new MLContext();
+            var model = PrepareModel(onnxModelFilePath, ctx);
+            return ctx.Model.CreatePredictionEngine<FloatInput, ProblematicOutputObj>(model);
+        }
+
+        [OnnxFact]
+        public void OnnxSequenceTypeWithouSpecifySequenceTypeTest()
+        {
+            Exception ex = Assert.Throws<Exception>(() => CreatePredictorWithPronlematicOutputObj());
```

Comment:
>CreatePredictorWithPronlematicOutputObj [](start = 58, length = 39)

A small typo here with Problematic

Reply:
yes, will do, thanks

---
In reply to: [330650375](https://github.com/dotnet/machinelearning/pull/4272#discussion_r330650375) [](ancestors = 330650375)